src/config/constants/nfts.ts

```diff
@@ -1,4 +1,16 @@
-import { Nft } from './types'
+import { Nft, NftSource, NftType } from './types'
+
+export const IPFS_GATEWAY = 'https://gateway.pinata.cloud'
+
+export const nftSources: NftSource = {
+  [NftType.PANCAKE]: {
+    address: {
+      56: '0xDf7952B35f24aCF7fC0487D01c8d5690a60DBa07',
+      97: '0x60935F36e4631F73f0f407e68642144e07aC7f5E',
+    },
+    identifierKey: 'image',
```

Comment:
Each `tokenURI` might (and probably will) have a different key to access the image (used in the identifier) so we keep track of it here.